src/main/java/htsjdk/samtools/util/SamLocusIterator.java

```diff
@@ -369,19 +428,29 @@ private boolean surpassedAccumulationThreshold() {
     }
 
     /**
+     * Check if cigar start with an insertion, ignoring softclip
+     * @param cigar the cigar element
+     * @return <code>true</code> if it start with an insertion; <code>false</code> otherwise
```

Comment:
...if the first operator to consume reference bases or be an insertion, is an insertion...